src/lib/utils/dissovled_oxygen_compensation.h

```diff
@@ -0,0 +1,94 @@
+#include <cmath>
+
+/**
+ * @brief Converts barometric pressure to a correction factor relative to standard sea-level pressure (760 mmHg).
+ * 
+ * @param pressure_mmhg Barometric pressure in mmHg (default: 760 mmHg).
+ * @return Correction factor to adjust DO saturation for pressure deviation.
+ * 
+ * Reference: 
+ * - https://water.usgs.gov/water-resources/memos/documents/WQ.2011.03.pdf
+ */
+double pressure_correction(double pressure_mmhg = 760.0) { return pressure_mmhg / 760.0; }
```

Comment:
The "Benson & Krauss" section actually gives a more complicated version of the pressure correction that uses both pressure and temperature and corrects for non-ideal gas behavior.
See equations (9), (10), and (11) in https://water.usgs.gov/water-resources/memos/documents/WQ.2011.03.pdf